.github/workflows/pytest-gpu.yml

```diff
@@ -48,72 +48,48 @@ jobs:
         ]
         include:
         - name: pytest-gpu-omp-nvidia
-          test_file: "tests/test_gpu_openmp.py"
-          env_file: "devito-ci-nvidia-openmp.env"
-          test_drive_cmd: "nvidia-smi"
-          arch: "clang"
-          platform: "nvidiaX"
-          language: "openmp"
-          tags: ["self-hosted", "gpu", "openmp", "clang"]
+          test_file: "tests/test_adjoint.py tests/test_gpu_common.py tests/test_gpu_openmp.py"
+          base: "devitocodes/bases:nvidia-clang"
+          tags: ["self-hosted", "gpu", "docker",]
+          flags: '--gpus all --rm --name testrun-clang-nvidia'
 
         - name: pytest-gpu-acc-nvidia
-          test_file: "tests/test_gpu_openacc.py"
-          env_file: "devito-ci-nvidia-openacc.env"
-          test_drive_cmd: "nvidia-smi"
-          arch: "nvc"
-          platform: "nvidiaX"
-          language: "openacc"
-          tags: ["self-hosted", "gpu", "openacc"]
+          test_file: "tests/test_adjoint.py tests/test_gpu_common.py tests/test_gpu_openacc.py"
+          base: "devitocodes/bases:nvidia-nvc"
+          tags: ["self-hosted", "gpu", "docker"]
+          flags: '--gpus all --rm --name testrun-nvc'
 
         - name: pytest-gpu-omp-amd
           test_file: "tests/test_gpu_openmp.py"
-          env_file: "devito-ci-amd-openmp.env"
-          test_drive_cmd: "rocm-smi"
-          arch: "aomp"
-          platform: "amdgpuX"
-          language: "openmp"
-          tags: ["self-hosted", "gpu", "openmp", "aomp"]
+          tags: ["self-hosted", "gpu", "docker", "openmp", "aomp"]
+          base: "devitocodes/bases:amd"
+          flags: "--network=host --device=/dev/kfd --device=/dev/dri --ipc=host --group-add video --group-add 109 --security-opt seccomp=unconfined --rm --name testrun-amd"
```

Comment:
lol 😬 

comment about where all these flags have been lifted from?

Reply:
I think this is simplified if we the current ubuntu LTR (22.1). But lets let the dust settle first.

Reply:
Only simplification will be `--group-add 109` removed all the rest will stay